alembic/testing/env.py

```diff
@@ -14,6 +14,24 @@
 from ..util.compat import u
 
 
+def _try_out(function, tries, ExceptionType = Exception, wait_time = 1.0):
```

Comment:
This can't be part of this patch.  if we wanted to add resiliency for running the tests on windows that has special software installed that prevents programs from running, that would be a totally different issue.  Also I'm -1 on that overall, I would recommend tuning your antivirus software to allow appropriate program activity.